src/components/home/partners/SecondaryPartners.jsx

```diff
@@ -130,19 +128,27 @@ const SmallScreenComponent = ({
         <Typography variant="body1" p={'5% 5% 5% 5%'}>
           {testimonial}
         </Typography>
-        <Typography variant="caption" p={'0 5% 8% 5%'}>
+        <Typography
+          variant="caption"
+          p={'0 5% 8% 5%'}
+          sx={{ fontWeight: 'bold', textAlign: 'center' }}
+        >
           {testimonialAuthor}
         </Typography>
-        {testimonialTwo ? (
-          <Typography variant="body1" display={'flex'} p={'5% 5% 5% 5%'}>
+        {testimonialTwo && (
+          <Typography variant="body1" p={'5% 5% 5% 5%'}>
             {testimonialTwo}
           </Typography>
-        ) : null}
-        {testimonialAuthorTwo ? (
-          <Typography variant="caption" p={'0 5% 10% 5%'}>
+        )}
+        {testimonialAuthorTwo && (
+          <Typography
+            variant="caption"
```

Comment:
Think the padding for `testimonialAuthorTwo` should be `0 5% 8% 5%` to match `testimonialAuthor`

Believe line 146 is reason why the is more space between the second testimonial and it's author vs the first.
<img width="148" alt="Screenshot 2023-12-19 at 4 57 17 PM" src="https://github.com/codeforpdx/codepdx_website/assets/71395931/5a230807-3e7a-43c9-9501-cd25094b92c6">
<img width="531" alt="Screenshot 2023-12-19 at 4 57 43 PM" src="https://github.com/codeforpdx/codepdx_website/assets/71395931/0e135445-b00b-45bc-84a8-045d94f69c04">